src/controller/python/test/test_scripts/base.py

```diff
@@ -178,6 +178,8 @@ def __init__(self, nodeid: int, paaTrustStorePath: str, testCommissioner: bool =
         self.logger = logger
         self.paaTrustStorePath = paaTrustStorePath
 
+        logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
I am assuming you want to remove this line

Reply:
Actually, I added that intentionally since we till date, have not been getting any verbose logs from the client-side in the Python test runs.